portal/models/qb_timeline.py

```diff
@@ -62,6 +62,50 @@ def qbd(self):
             recur_id=self.qb_recur_id, qb_id=self.qb_id)
 
 
+class AtOrderedList(list):
+    """Specialize ``list`` to maintain insertion order and ``at`` values
```

Comment:
`at values` seems too jargony. Something between "QB timeline snapshot creation datetime" and `at` would be ideal